src/test/java/teammates/test/cases/common/FeedbackQuestionAttributesTest.java

```diff
@@ -170,6 +174,45 @@ public void testValidate() {
 
         assertTrue(fq.isValid());
     }
+    
+    @Test
+    public void testGetQuestionDetails() {
+        FeedbackQuestionAttributes fq = new FeedbackQuestionAttributes();
+
+        ______TS("Text question: new Json format");
+        
+        fq.feedbackSessionName = "First Feedback Session";
+        fq.courseId = "CS1101";
+        fq.creatorEmail = "instructor1@course1.com";
+        fq.questionType = FeedbackQuestionType.TEXT;
+        fq.giverType = FeedbackParticipantType.NONE;
+        fq.recipientType = FeedbackParticipantType.RECEIVER;
+        fq.showGiverNameTo = new ArrayList<FeedbackParticipantType>();
+        fq.showGiverNameTo.add(FeedbackParticipantType.SELF);
+        fq.showRecipientNameTo = new ArrayList<FeedbackParticipantType>();
+        fq.showRecipientNameTo.add(FeedbackParticipantType.SELF);
+        fq.showResponsesTo = new ArrayList<FeedbackParticipantType>();
+        fq.showResponsesTo.add(FeedbackParticipantType.SELF);
+        fq.giverType = FeedbackParticipantType.TEAMS;
+        fq.recipientType = FeedbackParticipantType.TEAMS;
+        fq.showGiverNameTo = new ArrayList<FeedbackParticipantType>();
+        fq.showGiverNameTo.add(FeedbackParticipantType.RECEIVER);
+        fq.showRecipientNameTo = new ArrayList<FeedbackParticipantType>();
+        fq.showRecipientNameTo.add(FeedbackParticipantType.RECEIVER);
+        fq.showResponsesTo = new ArrayList<FeedbackParticipantType>();
+        fq.showResponsesTo.add(FeedbackParticipantType.RECEIVER);
```

Comment:
Instead of assigning these attributes manually, perhaps add a sample text question to `typicalDataBundle.json` which uses the new `questionMetaData` format and use that question to perform the assertion